apple/swift-docc
# Identify overloaded symbols

Bug/issue #, if applicable: rdar://117092401

## Summary

This is a PR to improve the presentation of overloaded symbols in Swift DocC related to this [forum post](https://forums.swift.org/t/improving-the-presentation-of-overloaded-symbols-in-swift-docc/67713). Specifically, this PR will:
1. identify overloaded symbols,
2. encode each individual overloaded symbol with the necessary information to display its sibling overloads

## Testing

Steps:
1. Build documentation for a target that has overloaded symbols. 
2. Open the .doccarchive and navigate to the Render JSON for one of the overloaded symbols. 
3. Check that the JSON contains an `otherDeclarations` array where each element contains an `identifier` and an array of `tokens`.

## Checklist
